### DISCUSSION TITLE
Write caching?

### DISCUSSION THREAD

**Discussion by farglabs** [readysettech/readyset#25] [General]
Without having tried out ReadySet yet, I have a question around inserts/update statements. From the deployment guide, it sounds like we should be able to switch our DB connection strings all over to point to our ReadySet instance. If we do that, it seems like this would cause quite a bit of extra latency on inserts/updates. How is this handled by the caching engine?

**Comment by jbensal** [comment]
Hey @farglabs 👋 ! 

That's a good question, you're right. Adding ReadySet to your application adds an extra network hop in to your query path.

For inserts/updates, the amount of latency really depends on your specific use case. ReadySet functions as a simple proxy, and we've seen the extra insert time be relatively small for users, so far. In the future, we can build in connection pooling to reduce this latency even further.

We're happy to hop on a [call](https://calendly.com/d/d5n-y44-mbg/office-hours-with-ready-set?month=2022-11&date=2022-11-18) to discuss this further, if that'd be of help!
